app/src/main/java/com/willowtree/vocable/settings/AddPhraseViewModel.kt

```diff
@@ -23,41 +23,6 @@ class AddPhraseViewModel : BaseViewModel() {
     private val liveShowPhraseAdded = MutableLiveData<Boolean>()
     val showPhraseAdded: LiveData<Boolean> = liveShowPhraseAdded
 
-    init {
-        populateMySayings()
-    }
-
-    private fun populateMySayings() {
-
-    }
-
-    fun deletePhrase(phrase: Phrase) {
-        backgroundScope.launch {
-            with(presetsRepository) {
-                deletePhrase(phrase)
-                val mySayingsCategory = getCategoryById(PresetCategories.USER_FAVORITES.id)
-                deleteCrossRef(
-                    CategoryPhraseCrossRef(
-                        mySayingsCategory.categoryId,
-                        phrase.phraseId
-                    )
-                )
-            }
-            populateMySayings()
-        }
-    }
-
-    fun updatePhrase(phrase: Phrase) {
-        backgroundScope.launch {
-            presetsRepository.updatePhrase(phrase)
-            populateMySayings()
-
-            liveShowPhraseAdded.postValue(true)
-            delay(PHRASE_UPDATED_DELAY)
-            liveShowPhraseAdded.postValue(false)
-        }
-    }
-
```

Comment:
how come this is getting removed? :D I am super curious!! :3

Reply:
So this wasn't being used, and for the database refactor this required some changes.
So I went ahead and removed it here so I don't have to change it.